linux/audacity

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+lib="${0%/*}/lib/audacity"
+
+export LD_LIBRARY_PATH="${lib}:${LD_LIBRARY_PATH}"
+export AUDACITY_MODULES_PATH="${lib}/modules"
+
+"${0%/*}/bin/audacity"
```

Comment:
Better:

```sh
exec "${0%/*}/bin/audacity" "$@"
```

This passes arguments to Audacity and replaces the shell process instead of starting a new one for Audacity (uses slightly fewer resources).